### ISSUE TITLE
Validate all turtle in the repo in CI

### ISSUE THREAD

**Issue by jimkont** [dbpedia/links#5]
We had a few syntax errors in some files that took time to identify & fix.
The idea is to create a TravisCI script that takes all rdf files in the repo and tries to validate them with rapper.
Travis has a new container based infrastructure were we can use normal bash scripts to do the validation
http://docs.travis-ci.com/user/migrating-from-legacy/

This approach will also validate all incoming pull requests

**Comment by jimkont** [comment]
also a way to ensure that the files are ordered would be good 

(although this might not be as easy)

**Comment by kurzum** [comment]
sort -u file.ttl > /tmp/file.ttl ; diff file.ttl f.ttl ??

**Comment by jimkont** [comment]
yes sure but we need a way to know which files needs to be tested for order ,we have other metadata rdf files in the repo

**Comment by kurzum** [comment]
Does the validate.sh fix this?

**Comment by jimkont** [comment]
no, but could be easily integrated now
